x/nft/types/query.pb.gw.go

```diff
@@ -20,7 +20,6 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/grpclog"
-	"google.golang.org/grpc/metadata"
```

Comment:
what are these changes in the generated pb?

Reply:
Not sure. Should I revert these? Integration tests seems to work fine with these changes.

Reply:
Maybe because of different `protoc` version on my machine?

Reply:
I'm not sure whether these metadata settings are anyhow critical for e.g. security headers or if one puts it behind LB. @yihuang do you know?

 maybe the best is to align it with how the generated code is in other projects? (cronos, ethermint, Cosmos SDK...)

Reply:
I've confirmed in Cosmos SDK. `.pb.gw.go` files in Cosmos SDK also doesn't have import for `google.golang.org/grpc/metadata` and any function call to `NewContextWithServerTransportStream`

Reply:
Merging this PR.